npm-shrinkwrap.json

```diff
@@ -1,62 +1,62 @@
 {
```

Comment:
Yeah it's pretty funky how npm on different machines can output different things to the shrinkwrap file.